mscore/data/shortcuts.xml

```diff
@@ -687,6 +687,10 @@
     <seq>P</seq>
     </SC>
   <SC>
+    <key>toggle-lyricseditor</key>
+    <seq>L</seq>
```

Comment:
Thanks very much. Interesting to see that the solution is to disable the action rather than the alphanumeric shortcut. (I.e. it still wouldn't be possible to display the piano keyboard even if the shortcut was manually set to "Ctrl+Alt+P" rather than just "P".) I suppose it doesn't matter since not all actions are useful in all modes, but the Lyrics Editor might be useful in Lyrics Entry Mode so perhaps I won't disable it and I'll pick a different shortcut instead, maybe Ctrl+Alt+L or an unused Function key.